docs/api/swagger.json

```diff
@@ -69,47 +75,927 @@
+          "type": "string",
+          "format": "uuid"
+        }
+      },
+      "groupId": {
+        "in": "path",
+        "name": "groupId",
+        "description": "Id from group",
+        "required": true,
+        "schema": {
+          "type": "string",
+          "format": "uuid"
+        }
+      },
+      "vocabularyId": {
+        "in": "path",
+        "name": "vocabularyId",
+        "description": "Id from vocabulary card",
+        "required": true,
+        "schema": {
+          "type": "string",
+          "format": "uuid"
+        }
+      },
+      "limit": {
+        "in": "query",
+        "name": "limit",
+        "description": "Limit defines the number of returned vocabulary from a request. Staged queries don't need a limit. Standart Limit: 100",
+        "required": true,
+        "schema": {
+          "type": "number"
+        }
+      },
+      "groups": {
+        "in": "query",
+        "name": "groups",
+        "description": "decides if groups are included in response",
+        "schema": {
+          "type": "boolean"
+        }
+      },
+      "stats": {
+        "in": "query",
+        "name": "stats",
+        "description": "decides if stats are included in response",
+        "schema": {
+          "type": "boolean"
+        }
+      },
+      "active": {
+        "in": "query",
+        "name": "active",
+        "description": "decides if this part gets included in querys",
+        "schema": {
+          "type": "boolean"
+        }
+      },
+      "activate": {
+        "in": "query",
+        "name": "activate",
+        "description": "decides if vocabs need to be activated before putting them into query drawers",
+        "schema": {
+          "type": "boolean"
+        }
+      },
+      "staged": {
+        "in": "query",
+        "name": "staged",
+        "description": "If true, all unactivated vocabs will be returned",
+        "schema": {
+          "type": "boolean"
+        }
+      },
+      "answer": {
+        "in": "query",
+        "name": "answer",
+        "description": "type in whether your answer was right or wrong",
+        "schema": {
+          "type": "boolean"
+        }
+      },
+      "progress": {
+        "in": "query",
+        "name": "progress",
+        "description": "If true, the progress will be returned",
+        "schema": {
+          "type": "boolean"
+        }
+      },
+      "name": {
+        "in": "query",
+        "name": "name",
+        "description": "decides if name gets returned",
+        "schema": {
+          "type": "boolean"
+        }
+      },
+      "code": {
+        "in": "query",
+        "name": "code",
+        "description": "decides if code gets returned",
+        "schema": {
+          "type": "boolean"
+        }
+      },
+      "nativeNames": {
+        "in": "query",
+        "name": "nativeNames",
+        "description": "decides if nativeNames gets returned",
+        "schema": {
+          "type": "boolean"
```

Comment:
Add a default values. Also in other parameters. See https://swagger.io/docs/specification/describing-parameters/ under `Default Parameter Values` section.